### ISSUE TITLE
New To The Web Industry?

### ISSUE THREAD

**Issue by Myussuf23** [kentcdodds/ama#89]
hi @kentcdodds , sorry for disturbing you, i am a student who want to learn about the web industry and i know web industry is changing very quickly that a new one may not easy to start now , so could u give me some tips about where and how to start? since u r an expert in this field your tips may benefit me . thanks friend.

**Comment by kentcdodds** [comment]
Hi @Myussuf23!
Thanks for the question!

> i know web industry is changing very quickly

Yes it is!

> a new one may not easy to start now

I actually think that the fast pace of the web is one of the things that makes it easy for newcomers to come in and make a difference really quickly. For example. I have only professionally been in the web industry for almost two years and I feel like I've been able to make a fairly good impact.

> so could u give me some tips about where and how to start?

I actually recently gave a talk at ForwardJS to answer that very question! You can find my slides and other info in my talks repo [here](https://github.com/kentcdodds/talks#zero-to-60-in-software-development-how-to-jumpstart-your-career). As soon as the video recording is made available, you'll find that there as well :-)

To summarize:

1. Learn
2. Build
3. Teach
4. Give freely
5. Don't be afraid to try new things and be willing to get rejected and fail
6. Be nice

I'll try to remember to post an update in here when the video is available :+1:

**Comment by Myussuf23** [comment]
thanks friend , i think this will help me.

**Comment by kentcdodds** [comment]
Hi there! My talk was published! I hope it's helpful to you :D https://youtu.be/-qPh6I2hfjw

**Comment by lukeschunk** [comment]
I watched it - it was awesome ^

**Comment by kentcdodds** [comment]
Thanks! :blush:

**Comment by kentcdodds** [comment]
Also related: I was on a podcast recently where I talked about this a bit: http://developingstory.netlify.com/tds-56-kent-c-dodds
